CHANGES.md

```diff
@@ -3,6 +3,8 @@ Change Log
 
 ### 1.23 - 2016-07-01
 
+* Added `heightReference` property to point, billboard and model entities that would allow them to be rendered relative to terrain.
+* Changed corridor, polygon and rectangle entities to conform to terrain by using a `GroundPrimitive` if they are filled and don't have `height` or `extrudedHeight`.
```

Comment:
Add a comment about the new Sandcastle example (and a link to where it will be on cesiumjs.org).